feature-configs/cn-ran-overlays/ran-profile/cluster-config/du-common/ptp/disable-chronyd.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    # Pay attention to the node label, create MCP accordingly
+    machineconfiguration.openshift.io/role: worker-cnf
+  name: disable-chronyd
+spec:
+  config:
+    ignition:
+      version: 2.2.0
+    systemd:
+      units:
+        - contents: |
+            [Unit]
+            Description=NTP client/server
+            Documentation=man:chronyd(8) man:chrony.conf(5)
+            After=ntpdate.service sntp.service ntpd.service
+            Conflicts=ntpd.service systemd-timesyncd.service
+            ConditionCapability=CAP_SYS_TIME
+
+            [Service]
+            Type=forking
+            PIDFile=/run/chrony/chronyd.pid
+            EnvironmentFile=-/etc/sysconfig/chronyd
+            ExecStart=/usr/sbin/chronyd $OPTIONS
```

Comment:
Is $OPTIONS being set somewhere?

Reply:
nope this is the file that exist in RHCOS I just copy the file here the important part is the `enable:false` all the other is just to don't change the file content

Reply:
understood, thanks